CHANGELOG.md

```diff
@@ -1,40 +1,81 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## 0.12.0 (July 29, 2022)

```

Comment:
If we're releasing this as the same time as vscode-microprofile, we should update this date to the 25th